components/service/glean/metrics.yaml

```diff
@@ -16,6 +16,8 @@ baseline:
       - baseline
     bugs:
       - 1497894
+    reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1512938#c3
```

Comment:
Should we make `reviews` required in the `metrics.yaml` schema (in glean-parser)?  That would have enforced these changes all along.  Or is that too heavy and there are legitimate use cases to leave these out?

Reply:
This is a good point. We should check back with @georgf on this. I feel like we really should, though, as it would prevent bad things from happening (i.e. unreviewed collections slipping in ) :)

Reply:
I like the forced `review` property too. As a developer I could at least link to an explanation/discussion why no review was needed.

Reply:
Perfect :) I filed [a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1518182) to discuss this! I'll flag Georg on it so that it doesn't slip through the cracks